server/views/pages/adjustments/tagged-bail/days.njk

```diff
@@ -66,7 +66,7 @@
       {{ govukButton({
         text: "Cancel",
         classes: "govuk-button--secondary",
-        href: "/" + prisoner.prisonerNumber
+        href: "/" + prisoner.prisonerNumber + "/tagged-bail/view"
```

Comment:
So this link will probably be conditional on whether or not your editing. The template and route are used for both. There is an `addOrEdit` variable used elsewhere.